ramls/check-in.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Check-in operation",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "Unique ID (generated UUID) of the check-in operation",
+      "$ref": "raml-util/schemas/uuid.schema"
+    },
+    "occurredDateTime": {
```

Comment:
I've been wondering about this, and whether we want a separate property for when the event was recorded.

What do you think?

Reply:
I think it should be important for reports, as Emma mentioned they wanted to be able to query logs for some time period, e.g. last month, year, etc.

Or do you mean something else?

Reply:
Apologies, I suggesting the idea that the `check in` event has both `occuredAt` for when the check in occurred and `recordedAt` for when that was recorded by the system.

Now that I think about it, I think the `metadata.createdDate` might be sufficient for the latter.

Reply:
Agree, the `metadata.createdDate` can be used as a `recordedAt` property.